.github/workflows/delete-stale-branches-preview.yml

```diff
@@ -0,0 +1,20 @@
+name: Delete stale visual snapshot branches Preview
+
+on:
+  # Allow workflow to be manually run from the GitHub UI
+  workflow_dispatch:
+jobs:
+  cleanup_visual_snapshot_branches:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete visual snapshot branches
+        uses: phpdocker-io/github-actions-delete-abandoned-branches@v2
+        with:
+          github_token: ${{ github.token }}
+          last_commit_age_days: 120
+          allowed_prefixes: visual-snapshot-update/pr-
+          ignore_branches: main,next,beta,15.x,13.x
+          # Disable dry run and actually get stuff deleted
+          dry_run: yes
+      - name: Get output
+        run: "echo 'Deleted branches: ${{ steps.delete_stuff.outputs.deleted_branches }}'"
```

Comment:
The "Delete visual snapshot branches" step needs `id: delete_stuff` for this to work.

Reply:
I would name it `delete_branches` though.

Reply:
Yes, my mistake...fixed it 👍